.github/ISSUE_TEMPLATE/join-contributors.md

```diff
@@ -17,3 +17,26 @@ Name | GitHub Handle | PyLadies Chapter | Team Role
 **Why do you want to join the PyLadies Project Communications team?**
 
 /cc @pyladies/communications
+
+**Issues or projects that I'm interested in helping out with**
+
+**Time zone**
+
+**Languages I speak**
+
+|Language | Level |
+| -- | -- 
+|Language 1| Native |
+|Language 2 | Intermediate|
+|Language 3 | Beginner|
+|Language 3 | Beginner |
+
+**Are you comfortable communicating in English?**
+
+**Monthly meetings**: Are you available to attend conference calls once a month? 
+- _NOTE:team members must be available for a monthly video call_
+
+**Involvement** : How frequently can you contribute to the team?
+- _NOTE: Team members are required to be active during a 3 month period_
```

Comment:
I think we might be able to say if you've missed 3 monthly meetings then you're not active...

Reply:
While that _seems_ fair, I am a little uncomfortable with it when I think about timezones, workload, internet bandwidth, and other factors.  Work and discussion can happen without meeting attendance, and making synchronous meetings at specific times can be difficult.  If I can't make meetings, what would be another way for me to still be "active" and meaningfully participate?

Things I'm thinking about:
- giving feedback/comments on one or more issues the group is working through
- opening issues on a relevant topic for the group
- submitting a PR or review, 
- having a substantial slack discussion on a topic, 
- attending one or more meetings in a three month period

Reply:
@albendz I agree that it's fair because a contributor should be able to join at least one in that period (if only for 5 or 10 mins)

@BethanyG you've also raised some good points because yes we need to understand the limitations/ challenges we all have and also don't want to discourage anyone from making a contribution (no matter how small it is).

 I think we should post a poll in the slack channel to get an idea of what everyone's thought's are. what do you both think ?

Reply:
I agree with @BethanyG , however, i think we should add "working on your assigned/chosen tasks" as one of the checklist.

Reply:
@AbigailMesrenyameDogbe @BethanyG @albendz I've created a poll in Slack, please vote when you have time